### PR TITLE
Feat/12 profile card in my profile

### PR DESCRIPTION
**Pull request by valmdz**
# Description

Implement  profile card component, with it's css and test for storybook. Created folders providing icons and pictures for tests. The code is set to be responsive.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

`npm run storybook`

 Go to: [http://localhost:3007]((http://localhost:3007))

profile_card1
profile_card2
profile_card3
profile_card4

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/users.controller.js [hunk 1]
```diff
@@ -5,26 +5,16 @@ const knex = require('../../config/db');
...
 const moment = require('moment-timezone');
 
-const getUsers = async () => {
-  try {
-    return await knex('users').select('users.id', 'users.title');
-  } catch (error) {
-    return error.message;
-  }
-};
+const getUsers = () => knex('users').select('*');
```
Comment (benna100) [inline src/server/api/controllers/users.controller.js:8]: This should not be part of the pr

File: src/server/api/controllers/users.controller.js [hunk 2]
```diff
@@ -5,26 +5,16 @@ const knex = require('../../config/db');
...
-      throw new Error(`incorrect entry with the id of ${id}`, 404);
-    }
-    return users;
-  } catch (error) {
-    return error.message;
+  const users = await knex('users')
+    .select('*')
+    .where({ id });
+  if (users.length !== 1) {
+    throw new Error(`incorrect entry with the id of ${id}`, 404);
```
Comment (benna100) [inline src/server/api/controllers/users.controller.js:15]: neither this

File: src/server/api/controllers/users.controller.js [hunk 3]
```diff
@@ -5,26 +5,16 @@ const knex = require('../../config/db');
...
   }
+  return users[0];
```
Comment (benna100) [inline src/server/api/controllers/users.controller.js:17]: neither this

File: src/server/api/controllers/users.controller.js [hunk 4]
```diff
@@ -47,7 +37,7 @@ const deleteUser = async (usersId) => {
 
 const createUser = async (body) => {
   await knex('users').insert({
-    title: body.title,
+    full_name: body.full_name,
```
Comment (benna100) [inline src/server/api/controllers/users.controller.js:40]: neither this

**Review by benna100 — CHANGES_REQUESTED** [review]
Remember to check your files changed to see what you will be adding to the codebase!

Have not looked at any of the other code

File: src/server/api/controllers/users.controller.js (hunk 1 above)
Comment (valmdz) [reply src/server/api/controllers/users.controller.js:8]: Yes I am aware of that and I will remove the the PR 82ac734 from here and just keep the other stuff but I believe I can do it until I am able to pull the merges from the users-controller-fix branch.

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ProfileCardComponent.styles.css';
+import iconLinkedin from '../../assets/images/icons/linkedin.svg';
+import iconWebsite from '../../assets/images/icons/website.svg';
+import iconGithub from '../../assets/images/icons/github.svg';
+import unknownUser from '../../assets/images/portraits/unknown-black.jpg';
+
+export const ProfileCardComponent = ({
+  profile_image_url,
```
Comment (benna100) [inline src/client/components/ProfileCardComponent/ProfileCardComponent.js:10]:
```suggestion
  profileImageUrl,
```

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  full_name,
```
Comment (benna100) [inline src/client/components/ProfileCardComponent/ProfileCardComponent.js:11]:
```suggestion
  fullName,
```

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  position,
+  linkedin,
+  website,
+  github,
+}) => {
+  return (
+    <div className="profile-card-whole">
+      <div className="top">
+        <div className="circle">
+          <img src={profile_image_url || unknownUser} alt="Portrait of user" />
```
Comment (benna100) [inline src/client/components/ProfileCardComponent/ProfileCardComponent.js:21]: Super nice!

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+        </div>
+        <h2 className="user-full-name">{full_name}</h2>
+        <p className="user-position">{position}</p>
+      </div>
+
+      <hr />
+      <div className="profile-card-social-links">
+        {linkedin && (
+          <div className="social-link">
+            <img src={iconLinkedin} className="social-icon" alt="icon" />
```
Comment (benna100) [inline src/client/components/ProfileCardComponent/ProfileCardComponent.js:31]:
```suggestion
            <img src={iconLinkedin} className="social-icon" alt="Social media icon" />
```

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+              href={`https://www.linkedin.com/in/${linkedin}`}
+              className="social-link-text"
+            >
+              {linkedin}
+            </a>
+          </div>
+        )}
+        {github && (
+          <div className="social-link">
+            <img src={iconGithub} className="social-icon" alt="icon" />
```
Comment (benna100) [inline src/client/components/ProfileCardComponent/ProfileCardComponent.js:42]: Better alt tag here!

**Review by benna100 — CHANGES_REQUESTED** [review]
Super nice!! Next time, do a seperate pr for each story. 

Why do we need the unknown blue, and unknown purple files?

There are a few changes

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js (hunk 9 above)
Comment (valmdz) [reply src/client/components/ProfileCardComponent/ProfileCardComponent.js:42]: ok

File: src/client/components/ProfileCardComponent/ProfileCardComponent.js (hunk 5 above)
Comment (valmdz) [reply src/client/components/ProfileCardComponent/ProfileCardComponent.js:10]: Bejamin, but in our database the names are given with the underscores and therefore I decided to go with the names that match the database

**Review by benna100 — APPROVED** [review]
